specs/notation-cli.md

```diff
@@ -127,6 +129,32 @@ OPTIONS:
    --help, -h                  show help (default: false)
 ```
 
+## login
+
+```console
+notation login --help
+NAME:
+   notation login - Provides credentials for authenticated registry operations
+
+USAGE:
+   notation login [options] [server]
+
+OPTIONS:
+   --username value, -u value    Username for registry operations
+   --password value, -p value    Password for registry operations
+   --help, -h                    Show help (default: false)
+
+POSITIONAL
+  <server>                       The registry URL for authentication
+
+GLOBAL ARGUMENTS
+   --plain-http                  Registry access via plain HTTP (default: false)
```

Comment:
@shizhMSFT, what is `--insecure` used for, and how common is the usage for `--ca-file` (I assume provides custom TLS trust store).

Reply:
`--insecure` is to skip the TLS cert check. Since `notation` is security oriented, we should not provide this functionality.

`--ca-file` is to specify the root CA cert to registry access. See https://github.com/oras-project/oras/issues/217

Reply:
@shizhMSFT, I was thinking this would be done with the `insecure registries` option in config.  
Does that still exist? 
Seems calling `--insecure` is too tedious. For the scenarios/instances where https isn't supported, such as IOT environments, I just assumed the config would solve the problem and not something that's done ad-hoc requiring the cli to support it.